internal/server/translator/translate.go

```diff
@@ -38,7 +38,7 @@ func Translate(
 	}
 
 	out := pb.TranslateResponse{}
-	mappings, err := mcf.ParseMapping(in.GetSchemaMapping(), "bq")
+	mappings, err := mcf.ParseMapping(in.GetSchemaMapping(), "datcom-store.dc_kg_latest")
```

Comment:
Can add a TODO to switch to the BQ analytics table once we switch over.  Then the public will have access to it...

Reply:
Done